src/Appwrite/Certificates/Adapter/LetsEncrypt.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Appwrite\Certificates\Adapter;
+
+use Appwrite\Certificates\Adapter;
+use Exception;
+use Utopia\App;
+use Utopia\CLI\Console;
+use Utopia\Database\DateTime;
+use Utopia\Logger\Log;
+use Utopia\System\System;
+
+class LetsEncrypt implements Adapter
+{
+    private string $email;
+
+    public function __construct()
+    {
+        $this->email = System::getEnv('_APP_EMAIL_CERTIFICATES', System::getEnv('_APP_SYSTEM_SECURITY_EMAIL_ADDRESS'));
+        if (empty($this->email)) {
+            throw new Exception('You must set a valid security email address (_APP_EMAIL_CERTIFICATES) to issue an SSL certificate.');
+        }
```

Comment:
typically we don't use environment variables inside the constructor. Since this is like a library and a library should be agnostic of application specifics like this env variable. This also makes it harder to decouple the library later on.  

It's be better if we pass these as arguments to the constructor.